makefile.shade

```diff
@@ -0,0 +1,136 @@
+default AUTHORS='Microsoft Open Technologies, Inc.'
+
+use-standard-lifecycle
+k-standard-goals
+
+default PROJECT_JSON="project.json"
+
+#api-check target="verify"
+    @{
+        var Configuration = E("Configuration");
+        var BASE_DIR = Directory.GetCurrentDirectory();
+
+        var apiCheckFullFramework = new FileInfo(Path.Combine(BASE_DIR, ".build\\ApiCheck\\tools\\net452\\ApiCheck.exe"));
+        var apiCheckCore = new FileInfo(Path.Combine(BASE_DIR, ".build\\ApiCheck\\tools\\netcoreapp1.0\\ApiCheck.dll"));
+
+        var directories = Directory.EnumerateDirectories(Path.Combine(BASE_DIR, ".\\src"));
+
+        var fullFrameworkMonikerPrefix = new string[] { "net45", "net46" };
+        var coreMonikerPrefix = new string[] { "netcoreapp", "netstandard" };
```

Comment:
IMO, it should be fine to look for `netstandard`. The only `netcoreapp` targeting things we ship are tools and I don't think we'll want to guarantee API compatibility for those.